src/renderers/opengl/gl_buffer.rs

```diff
@@ -1,3 +1,9 @@
+// Copyright (c) 2022 Emmanuel Gil Peyrot <linkmauve@linkmauve.fr>
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
```

Comment:
The code of this entire file has already been rewritten and is completely different from your implementation.
Your `Vbo` struct, which is what was used before, is an `enum` that becomes `Uploaded` once it is done, while `GlBuffer` is just a helper implemented with a builder pattern to construct a `NativeBuffer` from `glow`. It is not a rename of any of your code by any means.

As such, it is not subject to this license.

Reply:
Well, you can change things around any way you want, your code is still a derivative of the one I wrote, so not really no.

Reply:
It is not a modification of your code but a *complete rewrite*, so it's not a derivative.
It achieves the same result for the same goal, but it is not the same source code.
Checkout #22, I properly read the license and everything else falls under that term, but not this.